public/app/features/logs/components/LogRowMessage.test.tsx

```diff
@@ -13,6 +13,7 @@ const setup = (propOverrides?: Partial<ComponentProps<typeof LogRowMessage>>, ro
   const styles = getLogRowStyles(theme);
   const props: ComponentProps<typeof LogRowMessage> = {
     wrapLogMessage: false,
+    isExpanded: false,
```

Comment:
If we add this feature we will need a test for these changes. I can help with that.